pyglet/gui/widgets.py

```diff
@@ -204,25 +221,33 @@ def update_groups(self, order):
         self._sprite.group = OrderedGroup(order + 1, self._user_group)
 
     def on_mouse_press(self, x, y, buttons, modifiers):
+        if not self._enabled:
+            return
         if not self._check_hit(x, y):
```

Comment:
In the interest of keeping the module from getting too bloated, it would be better to combine the two "early out" checks. 
```py
if not self._enabled or not self._check_hit(x, y):
```

Reply:
Oh yikes did I actually do that?  That's embarrassing.  Will fix.